libs/application-generic/src/usecases/get-feature-flag/get-feature-flag.use-case.ts

```diff
@@ -17,4 +23,21 @@ export class GetFeatureFlag {
       defaultValue,
     });
   }
+
+  async getNumber(
+    command: GetFeatureFlagNumberCommand,
+  ): Promise<number | undefined> {
+    const value = process.env[command.key];
+    const defaultValue = prepareNumberStringFeatureFlag(
+      value,
+      command.defaultValue,
+    );
+
+    const result = await this.featureFlagsService.getWithFullContext({
+      ...command,
+      defaultValue,
+    });
+
+    return result === -1 ? command.defaultValue : result;
```

Comment:
this is something annoying on launchdarkly where we can not set the same default value for on/off state.
so i set off state to -1 and here i default to 100.

Reply:
Please add this as a comment.

Reply:
done